src/components/navbar/navbar.ts

```diff
@@ -92,96 +126,39 @@ export class MNavbar extends BaseNavbar implements MNavbarInterface {
             parseInt(elComputedStyle.paddingRight, 10) - parseInt(elComputedStyle.borderLeftWidth, 10) -
             parseInt(elComputedStyle.borderRightWidth, 10);
         if (width < listEl.clientWidth) {
-            let height: number = listEl.clientHeight;
+            this.computedHeight = listEl.clientHeight;
             this.hasScrolllH = true;
-            wrapEl.style.height = height + 40 + 'px';
-            this.$el.style.height = height + 'px';
-            this.scrollToSelectedElem();
+            wrapEl.style.height = this.computedHeight + 40 + 'px';
+            this.$el.style.height = this.computedHeight + 'px';
         } else {
             this.hasScrolllH = false;
             wrapEl.style.removeProperty('height');
             this.$el.style.removeProperty('height');
         }
     }
 
-    private scrollLeft(event: MouseEvent): void {
-        let wrapEl: HTMLElement = this.$refs.wrap as HTMLElement;
-        wrapEl.scrollLeft = wrapEl.scrollLeft - ((this.$el.clientWidth - 88) / 2.5);
-    }
-
-    private scrollRight(event: MouseEvent): void {
-        let wrapEl: HTMLElement = this.$refs.wrap as HTMLElement;
-        wrapEl.scrollLeft = wrapEl.scrollLeft + ((this.$el.clientWidth - 88) / 2.5);
-    }
-
-    public get model(): string {
-        return this.internalValue;
-    }
-
-    public set model(value: string) {
-        this.internalValue = this.disabled ? '' : value;
+    private get ComputedHeight(): string {
+        return this.computedHeight + 'px';
     }
 
-    private setItem(): void {
-        this.$children.forEach((child, index, arr) => {
-            if (index == 0 && arr.length >= 1) {
-                child['isFirst'] = true;
-            }
-            if (arr.length - 1 === index && arr.length > 1) {
-                child['isLast'] = true;
-            }
-        });
+    private get isSkinDark(): string {
+        return this.skin == 'dark' ? this.skin : 'light';
```

Comment:
Est-ce qu'on veut juste savoir si le skin est dark? Ou si on veut retourner la valeur de la skin?

Selon le nom de la computed, j'aurais tendance à juste faire
`return this.skin == 'dark'`.

Sinon, on retourne juste `this.skin`, ce qui enlève la nécessité d'avoir un getter. On n'a pas à trapper les skins invalides ici si on a déjà un Validator sur le Prop skin.